_build_cfg.yml

```diff
@@ -99,6 +99,8 @@ Topics:
     File: deployments
   - Name: Integrating external services
     File: integrating_external_services
+  - Name: Kubernetes
+    File: kubeconfig
```

Comment:
I think "Kubernetes" is probably too generic of a title for the `kubeconfig` topic. It's about the `.kubeconfig` file used to configure different profiles for the CLI (the `osc` command), so how bout something like "CLI Configuration", "CLI Setup", "Configuring the CLI", "CLI Configuration Profiles"... ? Whatever you land on, the top-level header (`=`) in the `kubeconfig.adoc` file can be updated to match (it currently reads ".kubconfig files").

Also, this topic could probably come right after the "Command Line Interface" topic in this list, since they're so related.